packages/maskbook/src/plugins/ITO/UI/ConfirmDialog.tsx

```diff
@@ -21,124 +19,110 @@ const useStyles = makeStyles((theme) =>
-                                            {item?.symbol}/{itoSettings.token?.symbol}
-                                        </Paper>
-                                    </Grid>
-                                    <Grid item xs={6}>
-                                        <Paper className={classes.data}>
-                                            {formatBalance(
-                                                new BigNumber(itoSettings.exchangeAmounts?.[index]),
-                                                item?.decimals ?? 0,
-                                                item?.decimals ?? 0,
-                                            )}
-                                        </Paper>
-                                    </Grid>
-                                </>
-                            )
-                        })}
+                {poolSettings?.exchangeTokens
+                    .filter((item, index) => item)
+                    .map((item, index) => {
+                        return (
+                            <>
+                                <Grid item xs={6}>
+                                    <Paper className={classes.label}>
+                                        {item?.symbol}/{poolSettings?.token?.symbol}
+                                    </Paper>
+                                </Grid>
+                                <Grid item xs={6}>
+                                    <Paper className={classes.data}>{poolSettings?.exchangeAmounts[index]}</Paper>
+                                </Grid>
+                            </>
+                        )
+                    })}
 
-                    <Grid item xs={6}>
-                        <Paper className={classes.label}>Allocation Per Wallet</Paper>
-                    </Grid>
-                    <Grid item xs={6}>
-                        <Paper className={classes.data}>{itoSettings.total}</Paper>
-                    </Grid>
+                <Grid item xs={6}>
+                    <Paper className={classes.label}>{t('plugin_ito_allocation_per_wallet')}</Paper>
+                </Grid>
+                <Grid item xs={6}>
+                    <Paper className={classes.data}>{poolSettings?.limit}</Paper>
+                </Grid>
 
-                    <Grid item xs={6}>
-                        <Paper className={classes.label}>Begin Times</Paper>
-                    </Grid>
-                    <Grid item xs={6}>
-                        <Paper className={classes.data}>{new Date(itoSettings.startTime).toLocaleString()}</Paper>
-                    </Grid>
+                <Grid item xs={6}>
+                    <Paper className={classes.label}>{t('plugin_ito_begin_times')}</Paper>
+                </Grid>
+                <Grid item xs={6}>
+                    <Paper className={classes.data}>
+                        {poolSettings?.startTime.toLocaleString().replace(/:00$/g, '')}
```

Comment:
?????

Reply:
remove second

Reply:
Date.prototype.toLocaleString()

Result is follow browser first-language.

Reply:
plz use `Intl.DateTimeFormat` or `date-fns/format`.

Reply:
CC @Jack-Works any idea?

Reply:
ok